po/README.md

```diff
@@ -1,3 +1,20 @@
 ### Translations
-You can translate Open Food Facts at https://translate.openfoodfacts.org. Please do not translate directly in the code, or your translations will be overwritten.
-If you have added or modified a string in the source file (the file with the .pot extension), you should update the crowdin-trigger branch from main/develop. This will trigger import within Crowdin, and 1 hour later, it will create a new pull request with added or modified strings in all language files, plus all added or modified translations.
+You can translate Open Food Facts at https://translate.openfoodfacts.org thanks to the Crowdin platform. Please do not translate directly in the code, or your translations will be overwritten.
+
+If you want to add a translatable string in the code:
+* modify the file with the `.pot` extension related to your context. Eg.
+  ```pot
+  msgctxt "name_of_your_string"
+  msgid "Hello world!"
+  msgstr ""
+  ```
+* identically modify the file `en.po` related to your context, to be sure at least an english string will be displayed before Crowdin send back all `.po` files (which can take several days)
```

Comment:
For en.po, we need the English translation in msgstr (which should have the same value as the msgid field)